cms/templates/js/move-xblock-list.underscore

```diff
@@ -27,7 +33,7 @@
                         </span>
                     <% } %>
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
-                    <span class="sr forward-sr-text"><%- forwardButtonSRText %></span>
+                    <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
```

Comment:
@catong @muhammad-ammar we really want this to be as concise as possible with as little cognitive load as possible.  It is repeated for every single item in the tree. "View child components" is 6 syllables.  How about "View children" ?  Users may not be familiar with Parent/Child terminology, that is my only concern, but my suggested text included it as well.  Thoughts?